### ISSUE TITLE
[apex] Fase Positive: Avoid logic in triggers rule is triggered for every trigger file

### ISSUE THREAD

**Issue by RedaBenh** [pmd/pmd#1373]
**Affects PMD Version:** 
6.6.0

**Rule:**
[AvoidLogicInTrigger](https://pmd.github.io/pmd-6.8.0/pmd_rules_apex_bestpractices.html#avoidlogicintrigger)

**Description:**
This rule is triggered for every trigger containing a least one block 

**Code Sample demonstrating the issue:**

```
trigger ProjectTrigger on Project__c (after insert) {
    if(trigger.isAfter && trigger.isInsert){
        ProjectTrigger_Handler.addDefaultMembers(trigger.new);
    }
}
```

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
run with Ant command

**Comment by rsoesemann** [comment]
This is no regression but how I have originally built and intended this rule.  No logic means no logic and an if block just is: logic.

**Comment by rsoesemann** [comment]
This is no regression but how I have originally built and intended this rule.  No logic means no logic and an if block just is: logic.

**Comment by adangel** [comment]
As stated by @rsoesemann , this is the intended behavior of the rule. I checked PMD 6.5.0 through 6.8.0 and 6.9.0-SNAPSHOT, we always report your snippet as a rule violation (so definitively not a regression).
I'm closing this issue, since it works as designed.
